### ISSUE TITLE
[Feature]: Display templated insert into command in empty tables

### ISSUE THREAD

**Issue by Andersama** [sqlitebrowser/sqlitebrowser#3817]
### Describe the new feature

Currently there is a very handy feature to copy an insert command which represents a row in a table with the shortcut `CTRL+ALT+C`

![image](https://github.com/user-attachments/assets/e1ba9d43-971f-48fd-bb95-dad7cae66447)

However in an empty table, the prompt appears:
![image](https://github.com/user-attachments/assets/7ba21158-215b-4215-9a78-91afb5248531)

Which would suggest it "does" something, but nothing occurs.

It would be very useful to have a shortcut and or context menu item which creates a prepared insert command. IE not one pre-filled with data from the row, but rather the insert command you'd actually use in your applications.

For example in my non-empty table using the shortcut I might get:
`INSERT INTO "main"."items" ("uuid", "item_name") VALUES ('35d94866-392d-4aab-86dc-098cd8b07511', 'an_item');`

We might imagine having these presented from any table:
`INSERT INTO "main"."items" ("uuid", "item_name") VALUES (@uuid, @item_name);`
`INSERT INTO "main"."items" ("uuid", "item_name") VALUES (:uuid, :item_name);`
`INSERT INTO "main"."items" ("uuid", "item_name") VALUES ($uuid, $item_name);`

This could also appear in the "Add New Record" prompt:
![image](https://github.com/user-attachments/assets/254a4594-6d80-48aa-8820-401bbea1df48)

**Comment by mgrojo** [comment]
Good idea, but I've implemented it in a slightly different way. There was already drag and drop support in the DB Schema dock for building select queries. I've added support for choosing insert statement too. Also for copying to the clipboard, and not only for drag and drop, but with the same text. You can try it in the latest nightly. There's a problem I still have to fix, though, related with saving the user election as settings.

**Comment by mgrojo** [comment]
I've fixed the problem with the settings and restructured the contextual menu. @Andersama, the next nightly build will be ready for testing.

**Comment by Andersama** [comment]
Took me a bit to figure out where the edit was. I was thinking more in the table browsing tab because at the moment you can select columns to create an insert command, but its prefilled with data from the row.

It may be worth making it its own shortcut, I don't deal with UI much, but I'd find it confusing that the behavior of `CTRL+C` changes based on a dropdown.
